front/src/modules/settings/workspace/components/NewObjectType.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { IconBox, IconCheck, IconDatabase, IconFileCheck } from '@/ui/icon';
+import { SoonPill } from '@/ui/pill/components/SoonPill';
+import { Tag } from '@/ui/tag/components/Tag';
+import { ThemeColor } from '@/ui/theme/constants/colors';
+
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: ${({ theme }) => theme.spacing(5)};
+  padding-bottom: ${({ theme }) => theme.spacing(10)};
+  padding-top: ${({ theme }) => theme.spacing(4)};
```

Comment:
Could you replace this with just a `margin-bottom`?

```suggestion
  margin-bottom: ${({ theme }) => theme.spacing(8)};
```